src/Symfony/Component/Workflow/Dumper/GraphvizDumper.php

```diff
@@ -120,8 +120,8 @@ protected function addTransitions(array $transitions)
     {
         $code = '';
 
-        foreach ($transitions as $place) {
-            $code .= sprintf("  transition_%s [label=\"%s\", shape=box%s];\n", $this->dotize($place['name']), $place['name'], $this->addAttributes($place['attributes']));
+        foreach ($transitions as $k => $place) {
```

Comment:
Super minor comment: instead of `$k`, it's more common to use `$i` for this kind of variable (`$k` is like "any generic key value" and `$i` is like the "auto-increment integer key value" used here).

Reply:
:+1: Fixed